blog/2015/03/getting-to-know-katy-huff.html

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+root: ../../..
+author: Amy Brown
+title: "Getting to Know: Katy Huff"
+date: 2015-03-15
+time: "19:30:00"
+category: ["Community"]
+---
+<!-- start excerpt -->
+<p>
+  This is the first in a series of posts about our contributors. We're posting
+  these so our community can get to know each other better. If you'd like to be
+  profiled, or you'd like to nominate another member, drop us a line.
+</p>
+<p>
+Our first profile is of <a href="http://software-carpentry.org/pages/team.html#huff.k">Katy Huff</a>, a member of our steering
+  committee, who is also on the finance subcommittee. We already know a little 
+  about Katy from her [Foundation Election nomination post](http://software-carpentry.org/blog/2015/01/scf-nomination-huff.html); now, read about her history with Software Carpentry and her surprising backup career plan.
+</p>
+<!-- end excerpt -->
+
+<h3>How did you become involved in SWC?</h3>
+<p>
+In 2008, when I was a graduate student in the University of Madison Wisconsin, my advisor, Paul P.H. Wilson, recommended that I watch <a href="https://nanohub.org/resources/1811">this presentation</a> that Greg put online while he was a professor at UT. Inspired by this, my friends and I, as The Hacker Within, created a workshop covering the topics that Greg emphasized as essential for scientists. Shortly thereafter, he found us when we popped up under search for Software Carpentry. A forever friendship was born.
+</p>
+
+<h3>What is your most memorable SWC moment?</h3>
+<p>
+Long ago, Greg sent me to Trieste, Italy to help teach Software Carpentry within a two week course for an international group of physicists. It was an intensive, advanced version of the Software Carpentry curriculum, but the students were dilligent and enthusiastic. Near the end of the course, my birthday came around and they all sang happy birthday in various languages to me over dinner. It was a blast but, most importantly, it felt very good to know I and Software Carpentry had been valuable to them.
+</p>
+
+<h3>If three phrases/words described you, what would they be?</h3>
+<p>
+Biking, vegetarian, nuclear engineer.
+</p>
+
+<h3>What research or work do you do?</h3>
+<p>
+I simulate various things involving nuclear energy. In my dissertation work, I focused on the systems analysis of nuclear material movement between facilities in the nuclear fuel cycle. In my postdoc, I have been more focused on multiphysics analysis of neutronics and thermal hydraulics in a very special advanced reactor design, the Pebble-Bed, Fluoride-salt-cooled, High-temperature Reactor (PB-FHR).
+</p>
+
+<h3>What programming languages and other technology do you regularly use?</h3>
+<ul>
+    <li>python (scipy, numpy, nose, pytables)</li>
```

Comment:
Can you replace this list with

~~~
<li>Python (SciPy, NumPy, nose, PyTables)</li>
<li>C++</li>
<li>Git</li>
<li>CMake</li>
<li>Sphinx</li>
<li>doxygen</li>
<li>GoogleTest</li>
~~~